.drone.yml

```diff
@@ -28,7 +28,7 @@ steps:
       - mkdir -p /tmp/${DRONE_BUILD_NUMBER} ; git rev-parse HEAD > /tmp/${DRONE_BUILD_NUMBER}/bbitest_hash
       - if [ ! -z $BBITEST_BRANCH ]; then git checkout $BBITEST_BRANCH; fi
     environment:
-      BBITEST_BRANCH: master
+      BBITEST_BRANCH: wrapt
```

Comment:
I just approved the bbitest repo, what might make more sense is to merge that first and then swap this back to master.

Reply:
yeah done